### PR TITLE
added client id support for kafka_franz input and output

### PR DESCRIPTION
**Pull request by ye11ow**
We are able to customize the client id

image

### CODE REVIEW

File: internal/impl/kafka/input_kafka_franz.go [hunk 1]
```diff
@@ -67,6 +67,9 @@ Finally, it's also possible to specify an explicit offset to consume from by add
 		Field(service.NewStringField("consumer_group").
 			Description("An optional consumer group to consume as. When specified the partitions of specified topics are automatically distributed across consumers sharing a consumer group, and partition offsets are automatically committed and resumed under this name. Consumer groups are not supported when specifying explicit partitions to consume from in the `topics` field.").
 			Optional()).
+		Field(service.NewStringField("client_id").
+			Description("An identifier for the client connection.").
+			Default("timeplus")).
```
Comment (ye11ow) [inline internal/impl/kafka/input_kafka_franz.go]: Maybe we should keep it `benthos` here in this repo and let Neutron to always pass `timeplus` or the confluent client ID?

File: internal/impl/kafka/output_kafka_franz.go [hunk 2]
```diff
@@ -48,6 +48,9 @@ This output often out-performs the traditional ` + "`kafka`" + ` output as well
 			Description("An optional explicit partition to set for each message. This field is only relevant when the `partitioner` is set to `manual`. The provided interpolation string must be a valid integer.").
 			Example(`${! meta("partition") }`).
 			Optional()).
+		Field(service.NewStringField("client_id").
```
Comment (ye11ow) [inline internal/impl/kafka/output_kafka_franz.go:51]: @jovezhong I added the client ID support for sink as well. Do we need it now?

File: internal/impl/kafka/input_kafka_franz.go (hunk 1 above)
Comment (zliang-min) [reply internal/impl/kafka/input_kafka_franz.go]: Yes, if we want to contribute back, that's a good idea.

Comment (ye11ow) [reply internal/impl/kafka/input_kafka_franz.go]: updated it to `benthos`

**Review by zliang-min — APPROVED** [review]

**Comment by jovezhong** [comment]
(Jove Github Bot) moved this ticket out of the GitHub project(up to 1200 tickets for one project).

**Pull request merged**